### ISSUE TITLE
Test results are poor compared to the training results

### ISSUE THREAD

**Issue by yifric** [NVIDIA/pix2pixHD#65]
I used the network on my dataset and the training results (web test images) were very good. However, the tested results were poor. 
I checked that i have the newest Torch version.
It may be noted that I made some changes:
- the PILL image mode of the label image is different. In your dataset the image mode was L, while I used RGB mode, In the instance image, you used L-mode while I used I-mode.
I used Label_nc=0 because of the label image format while in the cluster file creation I used label_nc=10 (otherwise the script doesn't run).

Do you have any idea what to do?

thanks

**Comment by tcwang0509** [comment]
It's basically overfitting. If your dataset is small, try reducing the network size (e.g. ngf).
